app.js

```diff
@@ -84,23 +73,52 @@ angular.module('wotApp', ['wotServices.wn8'])
 		.success(function(data, status, headers, config) {
 			$scope.playerTanks = data.data[$scope.accountId];
 
+			//console.log('playerTanks', JSON.stringify($scope.playerTanks));
```

Comment:
Please remove the commented lines.

Reply:
will do, was actually to see how many tanks we no longer use for stats as they were removed from the API.

And yes I did test, you will need to push to pages as well so it has the latest version :)

Reply:
here's a test version: http://balexandre.github.io/WotStats/ of this branch